Serilog.Sinks.Network/Sinks/TCP/TCPSocketWriter.cs

```diff
@@ -136,7 +138,7 @@ public TcpSocketWriter(Uri uri, int maxQueueSize = 5000)
                             try
                             {
                                 var messsage = Encoding.UTF8.GetBytes(entry);
-                                if (uri.Scheme.ToLower() == "tls")
+                                if (sslEnabled)
                                     ((SslStream)_stream).Write(messsage);
                                 else
                                     _stream.Write(messsage, 0, messsage.Length);

```

Comment:
@agoretsky @pauldambra what do you guys think of replace these Stream.Write() methods with Stream.WriteAsync()?